docs/02-for-app-authors/03-appdata-guidelines/01-quality-guidelines.md

```diff
@@ -0,0 +1,69 @@
+# Quality guidelines
+
+For the quick guidelines for the actual content (descriptions, screenshots), see the Appstream [docs](https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html).
+
+The following guidelines are optional, but recommended. We will likely promote apps that follow them in certain places.
+
+## General
+
+### No trademark violations
+
+Your app must not violate any trademarks. This includes the name, icon, and summary.
+
+### Managed by app author
+
+We recommend to verify your app on Flathub, but it’s not required.
+
+## App Icon
+
+### Icon size
+
+The icon must be at least 256x256px and be in PNG or SVG format.
+
+### Icon ratio
+
+The icon shouldn't fill less than 90% of the canvas or touch the edges.
```

Comment:
it must be square (appstream check this), even in SVG.

Reply:
```suggestion
The icon shouldn't fill less than 90% of the canvas or touch the edges. The icon must be square, including in SVG.
```

Reply:
This needs a bit more detail, and I'm not sure the precise 90% figure is helpful here.

```suggestion
The icon shouldn't fill too much or too little of the canvas. Avoid filling the entire canvas and having the icon touch the edges of the canvas (except with small protruding elements). Similarly, avoid thin shapes that are hard to see.
```

Reply:
I'll take @bertob version and will add that it has to be square to the `Icon size` part.

Reply:
Looks like this wasn't addressed and just merged as-is? I definitely wouldn't include 90% figure, and the sentence needs more nuance to be useful IMO.

Reply:
Yeah, missed it, as this was set to resolved by me for whatever reason? Probably due to a push.